recastjs/recast.idl

```diff
@@ -0,0 +1,141 @@
+interface rcConfig {
+    void rcConfig();
+    
+    /// The width of the field along the x-axis. [Limit: >= 0] [Units: vx]
```

Comment:
nit: convert to spaces instead of tabs for all files? there is a mixture right now in a few files